### ISSUE TITLE
code --version does not print to stdout

### ISSUE THREAD

**Issue by centigrade-thomas-becker** [Microsoft/vscode#14051]
- VSCode Version: 1.5.2
- OS Version: Windows 8

The command `code --version` does not print to stdout, instead it directly prints to screen. That makes it hard to capture and process the result.

Here is some example code:

```
var exec = require('child_process').exec;

exec('code --version', function (err, stdout, stderr) {
    console.log('stdout: ' + JSON.stringify(stdout));
    console.log('stderr: ' + JSON.stringify(stderr));
});
```

If I run this code then the version number will appear on the console, but the other output channels will be empty:

```
1.5.2 (66f37fd2a99eb9d628dd374d81d78835b410c39b)
stdout: ""
stderr: ""
```

**Comment by bpasero** [comment]
@centigrade-thomas-becker does it reproduce with 1.6?

**Comment by centigrade-thomas-becker** [comment]
It seems this was fixed in 1.6. 

Here's the output from 1.6.1:

```
stdout: "1.6.1\n9e4e44c19e393803e2b05fe2323cf4ed7e36880e\n"
stderr: ""
```
